libs/sr/formula/src/data/util/tag.ts

```diff
@@ -164,6 +164,9 @@ export const own = convert(ownTag, {
   et: 'own',
   dst: null,
 })
+export const semiOwn = convert(ownTag, {
```

Comment:
sorry last change
```suggestion
// `semiOwn` retains `dst`, so this can calculate a buff that comes from charA (`own`) but scales off charB stats (`target`)
export const semiOwn = convert(ownTag, {
```